pkg/services/oracle/neofs/neofs.go

```diff
@@ -119,18 +116,15 @@ func getRange(ctx context.Context, c *client.Client, addr *object.Address, ps ..
 	return checkUTF8(data)
 }
 
-func getHeader(ctx context.Context, c *client.Client, addr *object.Address) ([]byte, error) {
+func getHeader(ctx context.Context, c client.Client, addr *object.Address) ([]byte, error) {
 	obj, err := c.GetObjectHeader(ctx, new(client.ObjectHeaderParams).WithAddress(addr))
 	if err != nil {
 		return nil, err
 	}
-	msg := objectv2.ObjectToGRPCMessage(obj.ToV2()).Header
-	b := bytes.NewBuffer(nil)
-	err = new(jsonpb.Marshaler).Marshal(b, msg)
-	return b.Bytes(), err
+	return obj.ToV2().GetHeader().MarshalJSON()
```

Comment:
Isn't it still `ToV2` that we want to avoid? Or it'll be updated after nspcc-dev/neofs-api-go#301?

Reply:
yes, it will be updated after https://github.com/nspcc-dev/neofs-api-go/pull/301 and, i think, after `v1.27.1` patch release

now it is draft PR